source/_components/sensor.rest.markdown

```diff
@@ -153,3 +154,40 @@ sensor:
       User-Agent: Home Assistant REST sensor
 ```
 
+### {% linkable_title Fetch multiple JSON values and present them as attibutes %}
+
+This sample fetches a weather report from OpenWeatherMap.org, maps the
+resulting data into attributes of the RESTful sensor and then creates
+a set of template sensors that monitor the attributes and present the
+values in a usable form.
+
+```
+- platform: rest
+  name: OWM_report
+  json_attributes: main,weather
+  value_template: '{{value_json["weather"][0]["description"].title()}}'
+  resource: http://api.openweathermap.org/data/2.5/weather?zip=80302,us&APPID=VERYSECRETAPIKEY
+
+- platform: template
+  sensors:
+    owm_weather:
+      value_template: '{{states.sensor.owm_report.attributes.weather[0]["description"].title()}}'
+      icon_template: '{{"http://openweathermap.org/img/w/"+states.sensor.owm_report.attributes.weather[0]["icon"]+".png"}}'
+      entity_id: sensor.owm_report
```

Comment:
Please remove or update (the last two sensors) the `entity_id:` entries. It's advance feature and not needed to get the sensors running. Minimal samples are easier to understand for new users.

Reply:
OK. The inclusion of the `entity_id` was deliberate since without this the template gets re-evaluated periodically, which is rather inefficient. Since most of the pushback that I had in proposing this feature in the first place was to do with efficiency issues I thought I would make the example as efficient as possible!